tests/test_submit_tuxbuild.py

```diff
@@ -17,6 +19,15 @@ def setUp(self):
         self.squad = Squad()
         SquadApi.configure(url=self.testing_server, token=self.testing_token)
 
+        self.root_dir = os.path.join("tests", "data", "tux")
+        self.assertTrue(os.path.exists(self.root_dir))
+
+        self.build_dir = os.path.join(self.root_dir, "build-x86-gcc")
+        self.assertTrue(os.path.exists(self.build_dir))
+
+        self.buildset_dir = os.path.join(self.root_dir, "buildset-x86")
+        self.assertTrue(os.path.exists(self.buildset_dir))
```

Comment:
I think you can discard these 3 assertTrue above. I think you wanted to make sure you got the right rel path, but once you got it right, I think you don't need the asserts anymore.